docs/index.asciidoc

```diff
@@ -94,48 +94,53 @@ include::static/upgrading.asciidoc[]
 
 // Configuring Logstash
 
-:edit_url: https://github.com/elastic/logstash/edit/{branch}/docs/static/configuration.asciidoc
+:edit_url!:
```

Comment:
I can't find the definition of `:edit_url!:`, but assuming it is a macro that attaches to the following `include`, this is a _huge_ boon to readability.

Reply:
Good guess!  `:edit_url!:` clears the value so that the default is used.  Eventually, I will be able to pull all of these out, too! Thanks for the review.